### PR TITLE
Fixes #23265: Use consistent fonts in Rudder interface

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/23265

The current situation is messy, we use different font-families and don't provide them with Rudder, leading to inconsistent rendering. Lets use consistent fonts everywhere.

* To limit the effects of the changes, and have an experience consistent with the user platform, continue using system fonts but in a more consistent way.
* Font list based on bootstrap 5.3 (overriding our current bootstrap 3 with modern defaults).
* Use a global variable to only define font once and make future changes easier.

### CODE REVIEW

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

File: webapp/sources/rudder/rudder-web/src/main/webapp/secure/administration/apiManagement.html [hunk 1]
```diff
@@ -58,7 +58,7 @@
       #apiauthorization-app {}
 
       .rudder-template .dataTable tr>td.token .token-txt {
-        font-family: monospace;
+        font-family: font-family: var(--font-mono);;
```
Comment (ElaadF) [inline webapp/sources/rudder/rudder-web/src/main/webapp/secure/administration/apiManagement.html:61]: typo

**Comment by amousset** [comment]
PR updated with a new commit

File: webapp/sources/rudder/rudder-web/src/main/webapp/secure/administration/apiManagement.html (hunk 1 above)
Comment (amousset) [reply webapp/sources/rudder/rudder-web/src/main/webapp/secure/administration/apiManagement.html]: :+1:

File: webapp/sources/rudder/rudder-web/src/main/webapp/secure/administration/apiManagement.html [hunk 2]
```diff
@@ -58,7 +58,7 @@
       #apiauthorization-app {}
 
       .rudder-template .dataTable tr>td.token .token-txt {
-        font-family: monospace;
+        font-family: font-family: var(--font-mono);
```
Comment (ElaadF) [inline webapp/sources/rudder/rudder-web/src/main/webapp/secure/administration/apiManagement.html:61]: too many `font-family` again

**Comment by amousset** [comment]
PR updated with a new commit

File: webapp/sources/rudder/rudder-web/src/main/webapp/secure/administration/apiManagement.html (hunk 2 above)
Comment (amousset) [reply webapp/sources/rudder/rudder-web/src/main/webapp/secure/administration/apiManagement.html]: indeed :facepalm:

**Review by ElaadF — APPROVED** [review]

**Pull request merged**